packages/wrangler/src/dev/miniflare.ts

```diff
@@ -700,9 +700,8 @@ function buildAssetOptions(config: Omit<ConfigBundle, "rules">) {
 	if (config.assets) {
 		return {
 			assets: {
-				workerName: config.name,
```

Comment:
Does setting workerName here not remove the need for https://github.com/cloudflare/workers-sdk/pull/6835/files#diff-9ceb732889eaca4ea99209c2d4988f82a6ed60a853000e856d47fdcb687d67f2R1134-R1139

Reply:
We would also have to set this inside the vitest integration, which is a massive pain due to how the worker name is sometimes constructed with the test name as well. I think its easier to just set this entirely inside miniflare. Ideally I wouldn't expose this as an option for the asset plugin at all, I just can't figure out a nice way to pass something into a plugin without doing that.